app/js/controllers.js

```diff
@@ -134,6 +133,10 @@ function RoomCtrl($scope, $routeParams, $timeout, socket) {
     $scope.forceRevealDisable = (!$scope.forcedReveal && ($scope.votes.length < $scope.voterCount || $scope.voterCount === 0)) ? false : true;
 
     if ($scope.votes.length === $scope.voterCount || $scope.forcedReveal) {
+      $scope.votes = $scope.votes.sort(function(el1, el2) {
+        return $scope.cards.indexOf(el1.vote) - $scope.cards.indexOf(el2.vote);
+      });
+
```

Comment:
Hm. This might need a little more thought from my side. I like the idea, but it'll either look confusing on the reveal when the card you selected suddenly changes position in the stack, or potentially it would provide a "clue" that your estimate is too high or low before the reveal.

I'll have a think, but there might be something around a seperate section that lists the final scores away from the cards UI themselves - sorted, averaged, highest, lowest etc.

Reply:
I've just checked and it does not sort cards until revealed, so it won't give any hint. You are right, I might look a bit odd when your card moves on reveal. Maybe giving up on card flip animation would make it a better experience.

Reply:
@barthez
> It might be also implemented as additional action for room admin. One more button to sort cards on demand.

Agreed - this was the sort of direction I was thinking in - it would provide a means to allow other secondary actions as well. I'm not by any means attached to the card flipping animation, but I think there is an overall user experience to consider here, and allowing the cards to be sorted or grouped as a secondary action by the admin might help that.

Reply:
Added `Sort votes` button: 
![image](https://user-images.githubusercontent.com/151912/97803168-918b9880-1c48-11eb-8238-b32f05299f87.png)